app/src/main/java/com/onboarding/calculator/util/Util.java

```diff
@@ -0,0 +1,13 @@
+package com.onboarding.calculator.util;
+
+public class Util {
```

Comment:
this class may be named StringUtils or ConstantsUtils

Reply:
Solved!